sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -177,10 +179,6 @@ src_unpack() {
 
 src_prepare() {
 	# Flatcar: We don't have separate patches, so no patching code here.
-	#
-	# Flatcar: Use the resolv.conf managed by systemd-resolved.
-	sed -i -e 's,/run/systemd/resolve/stub-resolv.conf,/run/systemd/resolve/resolv.conf,' tmpfiles.d/etc.conf.m4 || die
```

Comment:
I think this will break all cases where 127.0.0.53 is not reachable because it's in a different namespace. I'll file a PR to revert this.